.github/scripts/verify_tag.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -e
+
+err() {
+    echo -e "\e[31m\e[1merror:\e[0m $@" 1>&2;
+}
+
+status() {
+    WIDTH=12
+    printf "\e[32m\e[1m%${WIDTH}s\e[0m %s\n" "$1" "$2"
+}
+
+REF=$1
+MANIFEST=$2
+
+if [ -z "$REF" ]; then
+    err "Expected ref to be set"
+    exit 1
+fi
+
+if [ -z "$MANIFEST" ]; then
+    err "Expected manifest to be set"
+    exit 1
+fi
+
+# strip preceeding 'v' if it exists on tag
+REF=${REF/#v}
+TOML_VERSION=$(cat $MANIFEST | dasel -r toml 'package.version')
```

Comment:
Where did you grab this script from? I thought the sway repo, but in the sway repo version this line is:
```suggestion
TOML_VERSION=$(toml get $MANIFEST package.version | tr -d '"')
```
Doesn't matter, but I wonder if one contains a fix the other is missing :thinking:

Reply:
fuel-core i think?

Reply:
https://github.com/FuelLabs/fuel-core/pull/160

Reply:
`fuel-core`. The one with dasel is faster since it's a binary. Hence requesting @Voxelot review!